ContinueEycaActivation/index.ts

```diff
@@ -39,19 +40,30 @@ export const index: AzureFunction = (
   message: unknown
 ): Promise<Failure | string> => {
   return fromEither(ContinueEycaActivationInput.decode(message))
-    .mapLeft(permanentDecodeFailure)
+    .mapLeft(e => permanentDecodeFailure(e))
```

Comment:
Why make `e` explicit ? 
```
.mapLeft(permanentDecodeFailure)
``` 
should suffice :)

Reply:
typo. I checked the type :D

Reply:
Done in [#30fb002](https://github.com/pagopa/io-functions-cgn/pull/27/commits/30fb002e32ae2a5436af016e2f7d52633bd4017c)